src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package task;
+
+import duke.DukeException;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    protected ArrayList<Task> taskArray;
+
+    protected String list;
+
+    protected int size;
+
+    /**
+     * Creates a TaskList.
+     * @param arr The ArrayList of stored task to be added.
+     */
+
+    public TaskList(ArrayList<Task> arr) {
+        this.taskArray = arr;
+        this.size = arr.size();
+    }
+
+    /**
+     * Creates an empty TaskList.
+     */
+
+    public TaskList() {
+        ArrayList<Task> task = new ArrayList<Task>(100);
+        this.taskArray = task;
+        this.size = 0;
+    }
+
+
+
+    public String printContent() throws DukeException {
+        String out = "";
+        try {
+            for (int i = 0; i < 2; i++) {
+                out = out + taskArray.get(i).toString() + "\n";
+            }
+        } catch (IndexOutOfBoundsException e) {
+            throw new DukeException("");
+        }
+        return out;
+    }
+
+
+    /**
+     * Gets the number of Tasks.
+     * @return The number of Tasks.
+     */
+    public int taskListSize() {
+        return size;
+    }
+
+
+    /**
+     * Adds the Task to the ArrayList.
+     * @param task The task to be added.
+     */
+    public void add(Task task) {
+        this.taskArray.add(task);
+        System.out.println("----------------------\n" + "Ok Solid you got this work to do:\n" +
```

Comment:
printing out any texts to the user (all user interactions) can be handled by the UI class